agent/config/agent.yaml

```diff
@@ -0,0 +1,26 @@
+# agent-integrations-next.yaml is intended to be used with
+# docker-compose.integrations-next.yaml. It demonstrates a config with
+# --enable-features=integrations-next and --config.expand-env, but does not
+# test absolutely all integrations.
+server:
+  log_level: info
+  http_listen_port: 12345
+
+metrics:
+  global:
+    external_labels:
+      origin_prometheus: yearnapi
+    scrape_interval: 1m
+    remote_write:
+    - url: ${REMOTE_WRITE}
+      basic_auth:
+        username: 313390
+        password: ${REMOTE_WRITE_PASSWORD}
+
+  configs:
+    - name: yearnapi
+      scrape_configs:
+        - job_name: yearnapi
+          static_configs:
+            - targets: ['api:3000']
```

Comment:
This assumes api is always run on port 3000 right?

Reply:
Yes it does, I'll change it so it uses the environment variable